### ISSUE TITLE
...torch/install/share/lua/5.1/cudnn/SpatialConvolution.lua:108: input has to contain: 16 feature maps, but received input of size: 1 x 19 x 29 x 29

### ISSUE THREAD

**Issue by jackieyung** [DmitryUlyanov/texture_nets#16]
When I ran the code:th stylization_process.lua -input_image data/readme_pics/kitty.jpg -model data/out/model.t7  -noise_depth 16  -save_path stylized.png.It just told me:...torch/install/share/lua/5.1/cudnn/SpatialConvolution.lua:108: input has to contain: 16 feature maps, but received input of size: 1 x 19 x 29 x 29.
The model.t7 is the one that produced by:th texture_train.lua -texture data/textures/red-peppers256.o.jpg -model_name pyramid -backend cudnn -num_iterations 1500 -vgg_no_pad true -normalize_gradients true -batch_size 15.
I don't know where is wrong.Could you give a solution?Thanks a lot.
Looking forward to your reply.
![snapshot1](https://cloud.githubusercontent.com/assets/13002561/17058660/23cc1a78-5054-11e6-9392-46c3d2e73fb8.png)

**Comment by DmitryUlyanov** [comment]
You ran texture synthesis instead of stylization. You need to run `stylization_train`.

**Comment by jackieyung** [comment]
Thanks a lot for your reply :).I just knew my problem.
But if I run stylization_train,I need to download the ILSVRC2012 validation set.It's too large to download right now.Could you send me a model that pre-trained?
Thank you very much.
